src/order/filter/basic.lean

```diff
@@ -1816,16 +1816,22 @@ lemma le_of_map_le_map_inj_iff {f g : filter α} {m : α → β} {s : set α}
   map m f ≤ map m g ↔ f ≤ g :=
 iff.intro (le_of_map_le_map_inj' hsf hsg hm) (λ h, map_mono h)
 
+lemma map_le_map {f g : filter α} {m : α → β} (hm : injective m) : map m f ≤ map m g ↔ f ≤ g :=
+by rw [map_le_iff_le_comap, comap_map hm]
```

Comment:
Isn't this just a weaker version of `le_of_map_le_map_inj_iff` above?

Reply:
(which should probably be named `map_le_map_of_inj_on` and be stated using `set.inj_on`)

Reply:
Renamed, removed `->` versions in favor of `iff`s.

Reply:
@eric-wieser Could you please have another look?